scripts/python/lib/argparse_gen.py

```diff
@@ -237,10 +239,29 @@ def get_args(parser_args=False):
         metavar='CONTAINER-NAME',
         help='Run all cluster deployment steps')
 
-    if parser_args:
-        return (
-            parser,
-            parser_setup, parser_config, parser_validate, parser_deploy)
+    # Check arguments
```

Comment:
Moving the argument check back in here again precludes sphinx from running correctly.